core/storage/email/gae_models.py

```diff
@@ -150,6 +196,16 @@ def check_duplicate_message(cls, recipient_id, email_subject, email_body):
         """Check for a given recipient_id, email_subject and cleaned
         email_body, whether a similar message has been sent in the last
         DUPLICATE_EMAIL_INTERVAL_MINS.
+
+		Args:
+            recipient_id: str. The user ID of the email recipient.
+            email_subject: str. The subject line of the email.
+            email_body: str. The HTML content of the email body.
+
+        Returns:
+            True if the message already existed in the messages.
```

Comment:
bool. Whether a similar message has been sent to the same recipient in the last DUPLICATE_EMAIL_INTERVAL_MINS.